Cython/Compiler/ModuleNode.py

```diff
@@ -107,6 +108,111 @@ def generate_c_code_config(env, options):
     ('__ge__', '__lt__'): (True, '', None),
 }
 
+class SharedUtilityExporter:
+    """
+    Class responsible for generating code responsible for importing and exporting shared utility functions.
+
+    This class is designed mark positions where the functions are being called by `call_import_code()`/`call_export_code()`.
+    The function calls and import/export functions are generated when `generate_exporting_functions()`
+    is called. This approach is needed because the list of the shared functions is known in the later stages
+    of compilation.
+    """
+    def __init__(self, mod_init_subfunction, scope, pos):
```

Comment:
`pos` always comes first for nodes, so let's stick to that here as well. (Obviously requires changing the callers also.)
```suggestion
    def __init__(self, pos, mod_init_subfunction, scope):
```